lua/cfc_random_spawn/module/sv_player_spawning.lua

```diff
@@ -66,13 +125,16 @@ function CFCRandomSpawn.handlePlayerSpawn( ply )
     if not ( ply and IsValid( ply ) ) then return end
     if IsValid( ply.LinkedSpawnPoint ) then return end
 
+    ply.isCFCRespawning = true
+
     CFCRandomSpawn.updateSpawnPointRankings( ply )
     local optimalSpawnPosition, optimalSpawnAngles  = CFCRandomSpawn.getOptimalSpawnPosition()
 
     ply:SetPos( optimalSpawnPosition )
     if optimalSpawnAngles then
         ply:SetEyeAngles( optimalSpawnAngles )
     end
+    ply.isCFCRespawning = false
```

Comment:
It's better to set this to to nil instead so the value isn't there when it isn't needed.